plugin-tester-java/src/main/java/example/myapp/helloworld/GreeterServer.java

```diff
@@ -39,20 +39,33 @@ public static void main(String[] args) throws Exception {
       // important to enable HTTP/2 in ActorSystem's config
       Config conf = ConfigFactory.parseString("akka.http.server.preview.enable-http2 = on")
         .withFallback(ConfigFactory.defaultApplication());
+
+      // Akka ActorSystem Boot
       ActorSystem sys = ActorSystem.create("HelloWorld", conf);
       Materializer mat = ActorMaterializer.create(sys);
 
+      // Instantiate implementation
       GreeterService impl = new GreeterServiceImpl(mat);
 
+      // Bind implementation to localhost:8080
       Http.get(sys).bindAndHandleAsync(
           GreeterServiceHandlerFactory.create(impl, mat),
-          ConnectWithHttps.toHostHttps("127.0.0.1", 8080).withCustomHttpsContext(serverHttpContext()),
+          // HTTP/2 servers are required to use TLS
```

Comment:
Perhaps link to https://doc.akka.io/docs/akka-http/current/server-side/http2.html#use-bindandhandleasync-and-https?

Reply:
Maybe that can be done in the context of removing the TLS code, @chbatey?